lib/membrane/rtp/outbound_packet_tracker.ex

```diff
@@ -138,21 +125,20 @@ defmodule Membrane.RTP.OutboundPacketTracker do
   end
 
   @impl true
-  def handle_other(:send_stats, _ctx, %{rtcp_output_pad: nil} = state) do
-    {:ok, state}
-  end
-
-  @impl true
-  def handle_other(:send_stats, _ctx, state) do
-    stats = get_stats(state)
-
-    actions =
-      %{state.ssrc => stats}
-      |> SenderReport.generate_report()
-      |> Enum.map(&Membrane.RTCP.Packet.serialize(&1))
-      |> Enum.map(&{:buffer, {state.rtcp_output_pad, %Membrane.Buffer{payload: &1}}})
-
-    {{:ok, actions ++ [redemand: state.rtcp_output_pad]}, %{state | any_buffer_sent?: false}}
+  def handle_other(:send_stats, ctx, state) do
+    if state.rtcp_output_pad && not ctx.pads[state.rtcp_output_pad].end_of_stream? do
```

Comment:
nitpick: what about assigning `state.rtcp_output_pad` to some variable that is shorter? We use it 3 times and it is quite lengthy